custom_typings/bower-config.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'bower-config' {
+  interface Config {
+    directory: string;
+  }
+  function read(cwd: string, overrides?: boolean): Config;
+}
```

Comment:
I've never done typescript before. I tried just adding `declare modules 'bower-config';` as instructed by the tool, but apparently the old version of tslint used in this project has a bug (since fixed), so I had to figure out something to make tslint and tsc happy.
I did just enough to declare the usage for this project, not sure if that is the normal methodology or if you expect the entire API to be declared. If so, I'll need a bunch of help figuring that out.